Tools/_sunrise/ci/sharding/test_shard_filter_tests.py

```
@@ -72,6 +132,51 @@ def test_builds_runsettings_with_escaped_filter(self):
+        groups, seconds = SHARD_FILTER.extract_groups(tests, timings, 2)
+
+        self.assertIn(("Content.Tests.Fixture", "Slow", tests[0]), groups)
+        self.assertEqual(seconds["Content.Tests.Fixture", "Slow", tests[0]], 10.0)
+
+    def test_parses_trx_duration(self):
+        self.assertEqual(
+            SHARD_FILTER._parse_trx_duration("00:01:02.5000000"),
+            62.5,
+        )
+
+    def test_requires_eighty_percent_of_timing_observations(self):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `test_requires_eighty_percent_of_timing_observations`

Add return type annotation: `None`

(ANN202)